app/src/main/java/com/isc/hermes/utils/PlacesSearchedManager.java

```diff
@@ -0,0 +1,23 @@
+package com.isc.hermes.utils;
+
+import android.content.Context;
+
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+/**
+ * This class manages the waypoints on the mapBox map.
+ */
+public class PlacesSearchedManager {
+
+    PlacesSearchedOnTheMap placesSearchedOnTheMap;
```

Comment:
Use encapsulation in your variable please `PlacesSearchedOnTheMap placesSearchedOnTheMap;`

Reply:
Done.